lib/inject.js

```diff
@@ -140,6 +138,25 @@ function injectPa11y(window, options, done) {
 		return false;
 	}
 
+	function hiddenAreasContainsElement(element, hiddenSelectors) {
+		var hiddenElements = window.document.querySelectorAll(hiddenSelectors);
+
+		for(var i = 0; i < hiddenElements.length; i++) {
+			if(element.isEqualNode(hiddenElements[i])) {
+				return true;
+			}
+
+			var parent = element.parentNode;
+			while (parent) {
+				if(parent.isEqualNode(hiddenElements[i])) {
```

Comment:
This fails the linting also for being too deeply nested, again refactor or ignore comments?